### PR TITLE
Switch all highlights to "fast" update

### PR DESCRIPTION
**Pull request by NiLuJe**
i.e., everything that does an invert
Plus a few other things that refresh small UI elements onTap
Re #3130 

Should hopefully make things feel a bit snappier :).

### CODE REVIEW

File: frontend/ui/widget/checkbutton.lua [hunk 1]
```diff
@@ -130,14 +130,14 @@ end
 function CheckButton:check()
     self:initCheckButton(true)
     UIManager:setDirty(self.parent, function()
-        return "partial", self.dimen
+        return "fast", self.dimen
```
Comment (NiLuJe) [inline frontend/ui/widget/checkbutton.lua]: Those two (partial -> fast) I'm less convinced about.
Not quire sure where they actually end up being used...

File: frontend/ui/widget/configdialog.lua [hunk 2]
```diff
@@ -81,7 +81,7 @@ function OptionTextItem:onTapSelect()
                     self.event, self.args,
                     self.events, self.current_item)
     UIManager:setDirty(self.config, function()
-        return "ui", self[1].dimen
+        return "fast", self[1].dimen
```
Comment (NiLuJe) [inline frontend/ui/widget/configdialog.lua]: Same as previously.
This "feels" right, though.

**Comment by NiLuJe** [comment]
Seems to work as intended besides these few comments.
The HLs themselves may even be a bit *too* snappy.

But I'm seeing A2 updates in strace, I'm happy. :D.

**Comment by Frenzie** [comment]
Pinging @poire-z 

lgtm ;-)

**Comment by NiLuJe** [comment]
Spoke too soon. Less than ideal on no-REAGL devices because of the aforementioned "partials on the full viewport" issue.

**Comment by NiLuJe** [comment]
Namely, instead of doing a small square A2 on a highlighted icon, it's done to the full viewport, which results in a pretty crappy-looking screen during the refresh.

That magically got optimized away on REAGL devices. (I was testing on a PW2, because it's what was plugged in -_-").

**Comment by poire-z** [comment]
I don't know real much about refreshes, and nothing about `fast`.
The only thing I can say about these refreshes (but that doesn't apply here then) is: don't use `partial` on popup windows, as it is annoying on kindle (I never used a kindle, but witnessed a bit of it when enabling half-broken REAGL on my GloHD). see #3230.

**Comment by NiLuJe** [comment]
What's (potentially) annoying with popups is specifically in the context of the reader, a partial on *closing* the popup risks leaving a ghost of itself in the text, that usually translates into a square of fainter text.

I think we took care of that a long time ago, though ;).

**Comment by NiLuJe** [comment]
https://github.com/NiLuJe/koreader/tree/uimanager-wip

Not sure this'll ever go anywhere, or if I'll ever have the time to figure it out, but, there ;).

**Comment by NiLuJe** [comment]
Huh. Another wonky thing:

Despite the H2O's viewport being (properly) set to `{x=0, y=11, w=1080, h=1429}`, I'm getting `{update_region={top=10, left=0, width=1080, height=1430}`... -_-"

**Pull request closed without merging**